cpp/CMakeLists.txt

```diff
@@ -62,6 +64,10 @@ endif()
 target_link_libraries(milvus-storage PUBLIC ${LINK_LIBS})
 target_include_directories(milvus-storage PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include ${CMAKE_CURRENT_SOURCE_DIR}/src)
 
+if(WITH_JNI)
+  target_include_directories(milvus-storage PRIVATE ${JNI_INCLUDE_DIRS})
```

Comment:
why target `milvus-storage` need include the `JNI_INCLUDE_DIRS` ?

and in L67, `JNI_INCLUDE_DIRS` have no defined.